src/content/developers/tutorials/downsizing-contracts-to-fight-the-contract-size-limit/index.md

```diff
@@ -146,3 +146,34 @@ function doSomething() { checkStuff(); }
 ```
 
 Those tips should help you to significantly reduce the contract size. Once again, I cannot stress enough, always focus on splitting contracts if possible for the biggest impact.
+
+### Use `bytes32` {#use-bytes32}
```

Comment:
Why 32 bytes specifically? If your data can fit into 8 bytes, or 4, you could use e.g. `bytes4` instead of padding out to 32 bytes. You could recommend using `bytesX` where X is the smallest number of bytes that can contain your data.